services/conservation_portfolios/src/util/dependency_injection_container.ts

```diff
@@ -0,0 +1,15 @@
+import Bottlejs from 'bottlejs';
+
+import Logger from './logger';
+import ErrorHandler from './error_handler';
+
+import ByTopicRoutes from '../routes/by_topic';
```

Comment:
In this case, can the routes be relative?

Reply:
they are relative